doc: Add Browan TBMS100 (#1228)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/models/browan-tbms100/_index.md

---
title: "Browan TBMS100"
description: ""
weight: 
---

{{< figure src="tbms100.png" alt="Generic Node Sensor Edition" class="float plain" >}}

The Browan [TBMS100](https://www.browan.com/product/motion-sensor-pir/detail) is a PIR motion sensor designed to detect occupancy or movement in homes or buildings for security and efficient facility management. It helps optimize the utilization of spaces like desks, conference rooms, and bathrooms.

<!--more-->

> # Onboarding to {{% tts %}}

Author: > We need it to be this size so it shows on the right hand menu (you can spot the difference if you deploy docs locally). Pls adjust other titles to this change

The right-hand side menu can only be visible if the page is zoomed out to 90% (for my computer). This applies to the entire site.

> # Onboarding to {{% tts %}}

Reviewer: hmm not sure why though, you should maybe report an issue for that? Because I can see it normally on 100%
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/65215579/da1ba16e-bdd4-488d-b264-2b159d3a9e76)

The provisioning information, such as **JoinEUI** and **DevEUI**, can be found on the back of the device. 

{{< figure src="tbms100-back.png" alt="TBMS100 provisioning information" >}}

{{< note "The sensitive information, the **AppKey**, is provided within an Excel sheet via email to the person who purchased it." />}}

The TBMS100 doesn't support onboarding with a QR code. You must proceed directly to the options [**Select the end device in the LoRaWAN Device Repository**]({{< ref "#using-the-lorawan-device-repository" >}}) or [**Enter end device specifics manually**]({{< ref "#manually" >}}).

## Using the LoRaWAN Device Repository

The [LoRaWAN® Device repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Browan TBMS100, which requires less information for registration with {{% tts %}}.

> In the **End device type** section, under **Input method**, select **Select the end device in the LoRaWAN Device Repository** option. The [Onboarding devices without QR codes]({{< ref "/devices/adding-devices/#onboarding-devices-without-qr-codes" >}}) section of the [Adding Devices]({{< ref "/devices/adding-devices/" >}}) page explains all the configurations you need to provide. However, the following are the TBMS100 specific configurations you want to include:

Reviewer: ```suggestion
In the **End device type** section, under **Input method**, select **Select the end device in the LoRaWAN Device Repository** option. The [Onboarding devices without QR codes]({{< ref "/devices/adding-devices/#onboarding-devices-without-qr-codes" >}}) section of the [Adding Devices]({{< ref "/devices/adding-devices/" >}}) guide explains this process in detail. Following details are TBMS100-specific:
```

Reviewer: Please don't use "all the configurations" in general. We are talking about one end device configuration at a time, with multiple parameters being set

- End device brand: **Browan Communication Incorp.**
- Model: **Motion Sensor**
- Hardware Ver.: **1.0**
- Firmware Ver.: **1.0.1**

> After providing all the configurations, select the **Register end device** button.

Reviewer: ```suggestion
After configuring your device, select the **Register end device** button.
```

{{< figure src="tbms100-device-repository.png" alt="Settings for registration through device repository" >}}

## Manually

> The TBMS100 also supports manual registration, but make sure all configurations are provided correctly.

Reviewer: remove

> In the **End device type** section, under **Input Method**, select **Enter end device specifics manually** option. The [Manually adding devices]({{< ref "/devices/adding-devices/manual/" >}}) and [Over the Air Activation (OTAA)]({{< ref "/devices/adding-devices/manual/otaa/" >}}) pages explain all the configurations you need to provide. However, the following are the TBMS100 specific configurations you want to include:

Reviewer: ```suggestion
In the **End device type** section, under **Input Method**, select **Enter end device specifics manually** option. The [Manually adding devices]({{< ref "/devices/adding-devices/manual/" >}}) and [Over the Air Activation (OTAA)]({{< ref "/devices/adding-devices/manual/otaa/" >}}) guides explain this procedure in detail. Following details are TBMS100-specific:
```

- LoRaWAN version: **LoRaWAN Specification 1.0.3**
- Regional parameters version: **RP001 Regional Parameters 1.0.3 revision A**

> After providing all the configurations, select the **Register end device** button.

Reviewer: ```suggestion
After configuring your device, select the **Register end device** button.
```

{{< figure src="tbms100-manually.png" alt="Settings for manual registration" >}}

> # Monitoring Live Data

Reviewer 2: ```suggestion
## Monitoring Live Data
```

According to the guidlines: 
> Since the title is a h1, everything in the content is at least h2 (##).

Author: Will fix it. Thanks.

> ## Monitoring Live Data

Reviewer: this can stay this size

In both registration options, after registering the device, you will be redirected to the **Overview** tab of the **End devices** page.

> {{< figure src="tbms100-overview-tab.png" alt="Overview tab" >}}

Reviewer: redundant, you can remove

> Then, select the **Live Data** tab to view all messages exchanged between the end device and the Network Server.

Reviewer: ```suggestion
When your device is registered, select the **Live Data** tab to view all messages exchanged between the end device and {{% tts %}}.
```

---
title: "Browan TBMS100"
description: ""
weight: 
---

{{< figure src="tbms100.png" alt="Generic Node Sensor Edition" class="float plain" >}}

The Browan [TBMS100](https://www.browan.com/product/motion-sensor-pir/detail) is a PIR motion sensor designed to detect occupancy or movement in homes or buildings for security and efficient facility management. It helps optimize the use of spaces like desks, conference rooms, and bathrooms, ensuring better utilization.

# Finding the Provisioning Information

The provisioning information, such as **JoinEUI** and **DevEUI**, can be found on the back of the device. 

{{< figure src="tbms100-back.png" alt="TBMS100 provisioning information" >}}

> {{< note "The **AppKey**, which is sensitive information, is provided within an Excel sheet." />}}

Author: Answer is pending from Lars

> {{< note "The **AppKey**, which is sensitive information, is provided within an Excel sheet." />}}

Author: Updated

# Onboarding to {{% tts %}}

> The TBMS100 doesn't support onboarding with a QR code. You must proceed directly to the options **Select the end device in the LoRaWAN Device Repository** or **Enter end device specifics manually**.

Reviewer: please refer to existing documentation here, for example you can link the guide for onboarding using a QR code and other sections

## Using the LoRaWAN Device Repository

The [LoRaWAN® Device repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Browan TBMS100, which requires less information for registration with {{% tts %}}.

In the **End device type** section, under **Input method**, select **Select the end device in the LoRaWAN Device Repository** option. Then select/enter the following parameters/settings:

- End device brand: **Browan Communication Incorp.**
- Model: **Motion Sensor**
- Hardware Ver.: **1.0**
- Firmware Ver.: **1.0.1**
- Profile (Region): choose **EU_863_870** for Europe or **US_902_928** for United States
- Frequency plan: choose the correct frequency plan for your TBMS100, **Europe 863-870 MHz (SF9 for RX2 - recommended)**, for example.

{{< figure src="tbms100-device-repository-1.png" alt="Settings for registration through device repository" >}}

In the **Provisioning information** section, enter the **JoinEUI** of the Join Server where the device has been pre-provisioned and then select the **Confirm** button.

Enter the **DevEUI** and **AppKey** into the relevant fields.

> Enter a new ID for your TBMS100 as the **End device ID** if you want to use a different one than the pre-filled End device ID.

Reviewer: ```suggestion
Enter a new ID for your TBMS100 as the **End device ID** if you want to use a different one than the pre-filled end device ID.
```

## Manually

The TBMS100 also supports manual registration, but make sure all parameters/settings are selected/entered correctly.

In the **End device type** section, under **Input Method**, select **Enter end device specifics manually** option. Then select/enter the following parameters/settings:

- Frequency plan: choose the correct frequency plan for your TBMS100, **Europe 863-870 MHz (SF9 for RX2 - recommended)**, for example.
- LoRaWAN version: **LoRaWAN Specification 1.0.3**
- Regional parameters version: **RP001 Regional Parameters 1.0.3 revision A**

Select **Show advanced activation, LoRaWAN class and cluster settings** to expand the section. Ensure that the default selection is **Over the Air Activation (OTAA)**.

{{< figure src="tbms100-manual-registration-1.png" alt="Settings for manual registration" >}}

In the **Provisioning information** section, enter the **JoinEUI** of the Join Server where the device has been pre-provisioned and then select the **Confirm** button.

Enter the **DevEUI** and **AppKey** into the relevant fields.

Enter a new ID for your TBMS100 as the **End device ID** if you want to use a different one than the pre-filled End device ID.

> Select the **Register end device** button.

Reviewer: All of these settings are not device-specific and are already explained in Devices section, so basically, you can just link that section and highlight some specifics only. I think this dilutes our content since we have too many places where we explain one thing over and over again

Author: @nejraselimovic I agree with you. I have more than 10 devices on my list for which I need to write documentation on how to add them to The Things Stack. However, it seems that I'll need to repeat the same information for all devices. As a suggestion, I propose creating a single page and listing the configuration settings in a table format. I am adding @KrishnaIyer to this loop.

{{< figure src="tbms100-manual-registration-p2.png" alt="Settings for manual registration " >}}

> # Viewing Live Data

Reviewer: ```suggestion
# Monitoring Live Data
```

In both registration options, after registering the device, you will be redirected to the **Overview** tab of the **End devices** page.

{{< figure src="tbms100-overview-tab.png" alt="Overview tab" >}}

> Then, select the **Live Data** tab to view all messages exchanged between the end device and the network server.

Reviewer: ```suggestion
Then, select the **Live Data** tab to view all messages exchanged between the end device and the Network Server.
```
